src/components/filters/AttributeFilter/AttributeFilter.tsx

```diff
@@ -96,25 +106,86 @@ export class AttributeFilter extends React.PureComponent<IAttributeFilterProps>
         );
     }
 
+    private isInverted() {
+        const { filter } = this.props;
+
+        return !AFM.isPositiveAttributeFilter(filter);
+    }
+
+    private getIdentifierOrUri() {
+        const { filter, identifier, uri } = this.props;
+        if ((filter && identifier) || (filter && uri) || (identifier && uri)) {
+            throw new Error("Don't use multiple attribute definitions");
+        }
+        if (identifier || uri) {
+            // tslint:disable-next-line:no-console
+            console.warn(
+                "Attribut definition by prop 'uri' or 'identifier' is deprecated. Please migrate to prop 'filter'. Please see documentation for [AttributeFilter component](https://sdk.gooddata.com/gooddata-ui/docs/attribute_filter_component.html)",
```

Comment:
Definition of attribute using 'uri' or 'identifier' is deprecated, use 'filter' property instead. Please see documentation of [AttributeFilter component](https://sdk.gooddata.com/gooddata-ui/docs/attribute_filter_component.html) for further details.

Reply:
fixed